TKGame/TKGame.cs

```diff
@@ -142,13 +147,16 @@ protected override void LoadContent()
             desktop = new Desktop();
             desktop.Root = VSP;
         }
-        protected override void Update(GameTime gameTime)
+        protected override async void Update(GameTime gameTime)
```

Comment:
i'm actually a bit surprised i haven't seen async in this project yet. it's pretty handy imo